go-controller/pkg/ovn/master_test.go

```diff
@@ -1394,26 +1394,11 @@ var _ = ginkgo.Describe("Default network controller operations", func() {
 			err = libovsdbops.DeleteLogicalRouterPorts(nbClient, lr, lrp)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
-			// Node delete will fail with Failed to delete node node1,
+			// Node delete should not fail
 			err = fakeClient.KubeClient.CoreV1().Nodes().Delete(context.TODO(), testNode.Name, metav1.DeleteOptions{})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 			// check the retry entry for this node
-			ginkgo.By("retry entry: old obj should not be nil, new obj should be nil")
```

Comment:
@ricky-rav with this change, delete will be capable of removing a partially removed node. Do you see this as a problem?

Reply:
no, it makes sense to me. The only thing I see here is that... we're removing a unit test for retrying a node delete and we're replacing it with a unit test for deleting a partially removed node. Can we have both, but in separate unit tests? In order to make the node delete operation fail, maybe we can just bring down nbdb, as we do for pods: https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/pods_test.go#L892-L896

Reply:
You are right, @ricky-rav . I will adpat the existing test, and create a new one to exercise this code